mongodb/terraform/data_mongodb_centos_ami.tf

```diff
@@ -11,5 +11,5 @@ data "aws_ami" "mongodb-centos" {
         values = ["hvm"]
     }
 
-    owners = ["${data.aws_caller_identity.current.account_id}"] 
+    owners = ["${var.image_owner}"] 
```

Comment:
Why do we need this change?

Do we need to export an environment variable to set this?